src/Parse/ParseObject.php

```diff
@@ -807,6 +806,7 @@ public static function destroyAll(array $objects, $useMasterKey = false)
      */
     private static function destroyBatch(array $objects, $useMasterKey = false)
     {
+        //@codingStandardsIgnoreStart
```

Comment:
Same thing here, you can break down the offending string being set on `path`. Looking it you can probably break it down fairly easily like so. Just be sure to remove the ignore start/end once done.
```php
$data[] = [
	'method' => 'DELETE',
	'path'   =>
		'/'.ParseClient::getMountPath().
		'classes/'.$object->getClassName().
		'/'.$object->getObjectId(),
];
```